components/tray/index.css

```diff
@@ -11,97 +11,87 @@ governing permissions and limitations under the License.
-
-  &.is-open {
-    /* Entry animations */
-    transition: transform
-        var(--mod-tray-entry-animation-duration, var(--spectrum-tray-entry-animation-duration))
-        cubic-bezier(0, 0, 0.4, 1)
-        var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay)),
-      opacity var(--spectrum-tray-entry-animation-duration, var(--mod-tray-entry-animation-duration))
-        cubic-bezier(0, 0, 0.4, 1)
-        var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay));
-    transform: translateY(0);
-  }
+	/* Default to full width when the viewport is in portrait orientation */
+	inline-size: 100%;
+	--mod-modal-max-width: 100%;
+	max-inline-size: 100%;
+
+	max-block-size: calc(100vh - var(--mod-tray-spacing-edge-to-tray-safe-zone, var(--spectrum-tray-spacing-edge-to-tray-safe-zone)));
+	margin-block-start: var(--mod-tray-spacing-edge-to-tray-safe-zone, var(--spectrum-tray-spacing-edge-to-tray-safe-zone));
+	padding-block-start: var(--mod-tray-top-to-content-area, var(--spectrum-tray-top-to-content-area));
+	padding-block-end: var(--mod-tray-bottom-to-content-area, var(--spectrum-tray-top-to-content-area));
+	padding-inline: var(--mod-tray-edge-to-content-area, var(--spectrum-tray-edge-to-content-area));
+	box-sizing: border-box;
+	overflow: auto;
+	outline: none;
+	border-radius: var(--mod-tray-corner-radius-portrait, 0) var(--mod-tray-corner-radius-portrait, 0) 0 0;
+
+	/* Start offset by the animation distance */
+	transform: translateY(100%);
+
+	/* Exit animations */
+	transition:
+		opacity var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration)) cubic-bezier(0.5, 0, 1, 1) var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-delay)),
+		visibility var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration)) linear calc(var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-delay)) + var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration))),
+		transform var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration)) cubic-bezier(0.5, 0, 1, 1) var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-delay));
+
+	background-color: var(--highcontrast-tray-background-color, var(--mod-tray-background-color, var(--spectrum-tray-background-color)));
+
+	&.is-open {
+		/* Entry animations */
+		transition:
+			transform var(--mod-tray-entry-animation-duration, var(--spectrum-tray-entry-animation-duration)) cubic-bezier(0, 0, 0.4, 1) var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay)),
+			opacity var(--spectrum-tray-entry-animation-duration, var(--mod-tray-entry-animation-duration)) cubic-bezier(0, 0, 0.4, 1) var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay));
```

Comment:
I'm struggling a little with this `opacity` block, so I might be totally wrong. But should the mod property come before the `--spectrum` property? 

from this:
`opacity var(--spectrum-tray-entry-animation-duration, var(--mod-tray-entry-animation-duration))`

to this: 
`opacity var(--mod-tray-entry-animation-duration, var(--spectrum-tray-entry-animation-duration))`

I noticed it was the only pairing in this section that's got the `spectrum` var first.